runtime/src/bank_forks.rs

```diff
@@ -234,6 +252,10 @@ impl BankForks {
         self[self.highest_slot()].clone()
     }
 
+    pub fn working_bank_with_scheduler(&self) -> &BankWithScheduler {
```

Comment:
The interface is slightly different for `working_bank` and `working_bank_with_scheduler`, with `working_bank` returning an owned (arc-wrapped) bank, and `working_bank_with_scheduler` returning only a reference.

This seems fine, I think probably the usage of `working_bank` **should** have been a reference - w/ the caller being responsible for cloning if ownership was needed.

Reply:
hmm, i'll pass for now... ;)

```
$ git diff
diff --git a/runtime/src/bank_forks.rs b/runtime/src/bank_forks.rs
index cd56ca195e2..6b37612ac12 100644
--- a/runtime/src/bank_forks.rs
+++ b/runtime/src/bank_forks.rs
@@ -245,6 +245,7 @@ impl BankForks {
         self.banks.values().map(|bank| bank.slot()).max().unwrap()
     }
 
+    #[deprecated()]
     pub fn working_bank(&self) -> Arc<Bank> {
         self[self.highest_slot()].clone()
     }

$ cargo clippy --all-targets |& grep "use of deprecated method" | wc -l
85
```